playbooks/informationals.md

```diff
@@ -18,11 +18,28 @@ If you're going to do it on-site in NYC, ask them to met you at Artsy, and give
 [La Colombe](https://code.dblock.org/2016/08/30/phone-screens-are-harmful-and-the-benefits-of-the-informational-interview.html)
 for an hour or so.
 
-Then do some back-and-forth to eventually find a time. Make a calendar event for the two of you.
+Then do some back-and-forth to eventually find a time. Make a calendar event for the two of you. Maybe even send
+them a link to this doc.
+
+#### Preparation
+
+For the person on the other end of the informational this is a big event. You may do these for Artsy with
+regularity, so it's important to always take time to do research on the person you're talking to and not get
+complacent that you can just sell Artsy.
+
+- Take the time to read through their CV
+- Google them, see what their public internet presence is
```

Comment:
We should specify this is to set the discussion context, and not for evaluating the candidate.